New Membership Guide (#1093)
CodingTrain/thecodingtrain.com: content/pages/guides/membership.md

- [YouTube Membership](https://youtube.com/thecodingtrain/join)
- [GitHub Sponsorshop](https://github.com/sponsors/shiffman)
- [Patreon Support](https://www.patreon.com/codingtrain)
- [Twitch Subscription](https://www.twitch.tv/subs/CodingTrainChooChoo): _twitch is presently not actively used but may be again in the future!_

Physical rewards (stickers and mystery gift) will be sent once a year, all together. Annoucements and address collection will happen in the members Discord channels. Climb aboard to receive the first mailing this summer 2023!

## What does this mean for existing members?

We promise this is good news for you! You’ll still receive the benefits of the tiers you’re in, regardless of what you’ve received in the past, but now with more a consistent system to receive the rewards every year.

If you're an existing member and missed any past rewards, please contact us at **[help@thecodingtrain.com](mailto:help@thecodingtrain.com)** to get back on track!

## Why these changes?

> After trying various membership models, we found the train was breaking down and often late. The support truly keeps the engines running here but we haven’t been able to support the current membership system adequately. However, we think that one annual mailing for those who are members rather than piecemeal things is something we can manage! At its core, being a member is about supporting the community, so thank you riding along and being a part of the community - Choo Choo!

Reviewer: "so thank you riding along" - missing the word `for`?

Author: Fixed!

> After trying various membership models, we found the train was breaking down and often late. The support truly keeps the engines running here but we haven’t been able to support the current membership system adequately. However, we think that one annual mailing for those who are members rather than piecemeal things is something we can manage! At its core, being a member is about supporting the community, so thank you riding along and being a part of the community - Choo Choo!

Author: ```suggestion
After trying various membership models, we found the train was breaking down and often late. The support truly keeps the engines running here but we haven’t been able to support the current membership system adequately. However, we think that one annual mailing for those who are members rather than piecemeal things is something we can manage! At its core, being a member is about supporting the community, so thank you for riding along and being a part of the community - Choo Choo!
```

> After trying various membership models, we found the train was breaking down and often late. The support truly keeps the engines running here but we haven’t been able to support the current membership system adequately. However, we think that one annual mailing for those who are members rather than piecemeal things is something we can manage! At its core, being a member is about supporting the community, so thank you for riding along and being a part of the community - Choo Choo!

Reviewer 2: ```suggestion
After trying various membership models, we found the merchandise delivery engine was breaking down and often late. Your support truly keeps the engines running here,  and we would like to do a better job of thanking you!  We think that one annual mailing rather than the current piecemeal approach is something we can manage. At its core, being a member is about supporting the Coding Train community, so we thank you for riding along - Choo Choo!
```

Author: @kchan2000 note these changes to this paragraph! We can mirror this in the instagram post, etc.

### Conductor $10 or more / month

- All the benefits of a Passenger, plus an annual mystery gift.

## Curious about Nature of Code?

You might have noticed that the number of live streams and new videos have slowed down in 2023. That's because I'm busy laying the tracks for the new version of the Nature of Code book. By supporting The Coding Train, you're also powering this project! And who knows, this book might be your mystery gift for 2024 when it's hot off the presses!

- [Nature of Code in progress](https://nature-of-code-2nd-edition.netlify.app/)
- [Nature of Code GitHub Repo](https://github.com/nature-of-code/noc-book-2023)

## How can I get my ticket?

You can support the Coding Train through any of the following platforms. Each may offer extra platform-specific perks (like emoji badges on YouTube), but everyone will get the rewards listed above!

- [YouTube Ticket Office](https://youtube.com/thecodingtrain/join)
- [GitHub Ticket Booth](https://github.com/sponsors/codingtrain)
- [Patreon Ticket Counter](https://www.patreon.com/codingtrain)
- [Twitch Ticket Stand](https://www.twitch.tv/subs/CodingTrainChooChoo) (_currently out of service, but might reopen!_)

> Your sticker pack and mystery gift will be dispatched once a year, all together. Keep an eye on the members-only Discord channels for announcements and instructions on where to send your goodies. Climb aboard to receive the first mailing this summer 2023!

Reviewer 2: ```suggestion
Your sticker pack and mystery gift will be dispatched once a year, all together. Keep an eye on the members-only Discord channels for announcements and instructions on how to receive your goodies. Climb aboard to receive the first mailing this summer 2023!
```
